recipes/parallelproj/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
```

```suggestion
build:
  run_exports:
    - {{ pin_subpackage( name|lower, max_pin='x.x') }}
```
Because this is a shared object with headers, you must include a run_export which pins according to the *ABI guarantees* of your library.